source/blog/2017-07-11-fedora-atomic-26-release.html.md

```diff
@@ -0,0 +1,551 @@
+* selinux-policy-targeted 3.13.1-225.18.fc25.noarch -> 3.13.1-259.fc26.noarch
+* setools-console 3.3.8-12.fc25.x86_64 -> 4.1.0-3.fc26.x86_64
+* setup 2.10.4-1.fc25.noarch -> 2.10.5-2.fc26.noarch
+* shadow-utils 2:4.2.1-11.fc25.x86_64 -> 2:4.3.1-3.fc26.x86_64
+* shared-mime-info 1.8-1.fc25.x86_64 -> 1.8-2.fc26.x86_64
+* socat 1.7.3.2-1.fc25.x86_64 -> 1.7.3.2-2.fc26.x86_64
+* sos 3.4-1.fc25.noarch -> 3.4-1.fc26.noarch
+* sqlite-libs 3.14.2-1.fc25.x86_64 -> 3.19.1-1.fc26.x86_64
+* sssd-client 1.15.2-5.fc25.x86_64 -> 1.15.2-5.fc26.x86_64
+* strace 4.17-1.fc25.x86_64 -> 4.17-1.fc26.x86_64
+* sudo 1.8.20p2-1.fc25.x86_64 -> 1.8.20p2-1.fc26.x86_64
+* system-python 3.5.3-6.fc25.x86_64 -> 3.6.1-8.fc26.x86_64
+* system-python-libs 3.5.3-6.fc25.x86_64 -> 3.6.1-8.fc26.x86_64
+* systemd 231-17.fc25.x86_64 -> 233-6.fc26.x86_64
+* systemd-bootchart 231-2.fc25.x86_64 -> 231-3.fc26.x86_64
+* systemd-container 231-17.fc25.x86_64 -> 233-6.fc26.x86_64
+* systemd-libs 231-17.fc25.x86_64 -> 233-6.fc26.x86_64
+* systemd-pam 231-17.fc25.x86_64 -> 233-6.fc26.x86_64
+* systemd-udev 231-17.fc25.x86_64 -> 233-6.fc26.x86_64
+* tar 2:1.29-3.fc25.x86_64 -> 2:1.29-4.fc26.x86_64
+* tcp_wrappers 7.6-83.fc25.x86_64 -> 7.6-85.fc26.x86_64
+* tcp_wrappers-libs 7.6-83.fc25.x86_64 -> 7.6-85.fc26.x86_64
+* tcpdump 14:4.9.0-1.fc25.x86_64 -> 14:4.9.0-2.fc26.x86_64
+* teamd 1.27-1.fc25.x86_64 -> 1.27-1.fc26.x86_64
+* timedatex 0.4-2.fc24.x86_64 -> 0.4-3.fc26.x86_64
+* tmux 2.2-3.fc25.x86_64 -> 2.3-2.fc26.x86_64
+* trousers 0.3.13-6.fc24.x86_64 -> 0.3.13-7.fc26.x86_64
+* trousers-lib 0.3.13-6.fc24.x86_64 -> 0.3.13-7.fc26.x86_64
+* tzdata 2017b-1.fc25.noarch -> 2017b-1.fc26.noarch
+* userspace-rcu 0.9.2-2.fc25.x86_64 -> 0.9.3-2.fc26.x86_64
+* ustr 1.0.4-21.fc24.x86_64 -> 1.0.4-22.fc26.x86_64
+* util-linux 2.28.2-2.fc25.x86_64 -> 2.29.1-2.fc26.x86_64
+* which 2.21-1.fc25.x86_64 -> 2.21-2.fc26.x86_64
+* xfsprogs 4.9.0-1.fc25.x86_64 -> 4.10.0-1.fc26.x86_64
+* xkeyboard-config 2.20-2.fc25.noarch -> 2.21-1.fc26.noarch
+* xz 5.2.2-2.fc24.x86_64 -> 5.2.3-2.fc26.x86_64
+* xz-libs 5.2.2-2.fc24.x86_64 -> 5.2.3-2.fc26.x86_64
+* yajl 2.1.0-5.fc24.x86_64 -> 2.1.0-6.fc26.x86_64
+* zlib 1.2.8-10.fc24.x86_64 -> 1.2.11-2.fc26.x86_64
+
+Downgraded:
+
+* dbus 1:1.11.14-1.fc25.x86_64 -> 1:1.11.12-1.fc26.x86_64
+* dbus-libs 1:1.11.14-1.fc25.x86_64 -> 1:1.11.12-1.fc26.x86_64
+* libgcrypt 1.7.8-1.fc25.x86_64 -> 1.7.7-1.fc26.x86_64
+* skopeo 0.1.22-1.git5d24b67.fc25.x86_64 -> 0.1.19-1.dev.git0224d8c.fc26.x86_64
+* skopeo-containers 0.1.22-1.git5d24b67.fc25.x86_64 -> 0.1.19-1.dev.git0224d8c.fc26.x86_64
+* vim-minimal 2:8.0.679-1.fc25.x86_64 -> 2:8.0.662-1.fc26.x86_64
+
+Removed:
+
+* libgudev-230-3.fc24.x86_64
+* newt-0.52.19-2.fc25.x86_64
+* newt-python3-0.52.19-2.fc25.x86_64
+* plymouth-0.9.3-0.6.20160620git0e65b86c.fc25.x86_64
+* plymouth-core-libs-0.9.3-0.6.20160620git0e65b86c.fc25.x86_64
+* plymouth-scripts-0.9.3-0.6.20160620git0e65b86c.fc25.x86_64
+* ppp-2.4.7-9.fc24.x86_64
+
+Obsolesced due to name changes:
```

Comment:
We had to do a quick check.  But as-is, the "removed" section was fairly misleading.